rpmsign.c

```diff
@@ -166,8 +184,9 @@ int main(int argc, char *argv[])
 	argerror(_("no arguments given"));
     }
 
-#ifdef WITH_IMAEVM
-    if (fileSigningKey && !(sargs.signflags & RPMSIGN_FLAG_IMA)) {
+#if defined(WITH_IMAEVM) || defined(WITH_FSVERITY)
+    if (fileSigningKey &&
+	!(sargs.signflags & (RPMSIGN_FLAG_IMA | RPMSIGN_FLAG_FSVERITY))) {
 	argerror(_("--fskpath may only be specified when signing files"));
```

Comment:
The actual {} block ends on the same indentation level as the condition, which makes it unnecessarily hard to read. Either:
- use a helper variable to shorten the condition to fit on one line
- further indent the line continuation to make it stand out
- put the opening { on a line of its own

With good naming, the helper variable approach typically yields by far the most readable code, but it depends.
Similar indentation problems are present elsewhere in the patch(es) too, please check out for them.

Reply:
I like this and introduced a flags_sign_files() helper. It was applicable in two places, so it made the code a lot easier to read.